### PR TITLE
Add decoding of StartupLocality

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
Signed-off-by: Mike Spreitzer <mspreitz@us.ibm.com>

### CODE REVIEW

**Comment by idesai** [comment]
@williamcroberts please can you approve to run the workflows, I tried at my end and I end up at a 404 page.

**Comment by williamcroberts** [comment]
> @williamcroberts please can you approve to run the workflows, I tried at my end and I end up at a 404 page.

I got a 500

**Comment by williamcroberts** [comment]
@idesai try testing and merging manually?

File: lib/efi_event.h [hunk 1]
```diff
@@ -187,4 +187,19 @@ typedef struct {
     // UINT8 OptionalData[];
 } PACKED EFI_LOAD_OPTION;
 
+/*
+ * EV_NO_ACTION_STRUCT is the structure of an EV_NO_ACTION event.
+ * Described in TCG PCClient FPF section 9.4.5.
```
Comment (MikeSpreitzer) [inline lib/efi_event.h:192]: Oh, ugh, it's PFP not FPF.  https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PFP_r1p05_05_3feb20.pdf

Comment (idesai) [reply lib/efi_event.h:192]: @MikeSpreitzer, please can you then fix the typo :)

**Comment by idesai** [comment]
> @idesai try testing and merging manually?

Looks like that would be the way.
PS: I had another PR waiting approval for running the CI workflows. But I could approve that and it launched without issues.
I am wondering what's special on this PR :)

File: lib/efi_event.h (hunk 1 above)
Comment (MikeSpreitzer) [reply lib/efi_event.h]: fixed

**Comment by MikeSpreitzer** [comment]
The force-push to 4b6fe2f2 adds the overlooked DCO signoff in the second commit.

**Pull request merged**